package-lock.json

```diff
@@ -8728,6 +8728,12 @@
       "integrity": "sha1-SMptcvbGo68Aqa1K5odr44ieKwk=",
       "dev": true
     },
+    "image-size": {
```

Comment:
unsure why this is being committed if there is no change to package.json

Reply:
Yeah i think i added that file on accident. My bad.